make/test.mk

```
@@ -153,7 +155,8 @@ endif
 
 .PHONY: clean-e2e-resources
 clean-e2e-resources:
-	oc get catalogsource --output=name -n openshift-marketplace | grep "toolchain-operator" | xargs --no-run-if-empty oc delete -n openshift-marketplace
-	oc get subscription --output=name -n ${TOOLCHAIN_NS} |  grep "toolchain-operator" | xargs --no-run-if-empty oc delete -n ${TOOLCHAIN_NS}
-	oc get subscription --output=name -n openshift-operators |  grep "openshift-pipelines-operator" | xargs --no-run-if-empty oc delete -n openshift-operators
-	oc delete project toolchain-workspaces --timeout=10s 2 > /dev/null || true
+	oc get catalogsource --output=name -n openshift-marketplace | grep "toolchain-operator" | xargs oc delete -n openshift-marketplace
```

to ignore failures when --no-run-if-empty is removed
```suggestion
	-oc get catalogsource --output=name -n openshift-marketplace | grep "toolchain-operator" | xargs oc delete -n openshift-marketplace
```